package.json

```diff
@@ -75,7 +75,7 @@
     "style-loader": "0.16.1"
   },
   "dependencies": {
-    "@bufferapp/components": "0.5.17",
+    "@bufferapp/components": "0.5.18",
```

Comment:
I _think_ this might cause a conflict as I've just merged a patch into `master` and bumped us up to `0.5.19`. 😅 So sorry for any extra work there!